cmake-docker-environment/lab-files/targets-example/CMakeLists.txt

```diff
@@ -1,13 +1,12 @@
 cmake_minimum_required(VERSION 3.9.1)
 project(songTest VERSION 1.0.0)
 
-add_library(firstDay SHARED firstDay/firstDay.cpp)
+add_library(firstDay firstDay/firstDay.cpp)
+target_include_directories(firstDay PUBLIC ${CMAKE_CURRENT_LIST_DIR}/firstDay/include)
 
-add_library(secondDay SHARED secondDay/secondDay.cpp)

```

Comment:
It's a good practice to avoid forcing shared compilation if it is not strictly needed. That way, the user can choose to build shared/static via command line (see how to do it in the readme)